tests/visco_plastic_phases_viscosity.prm

```diff
@@ -120,4 +120,4 @@ subsection Solver parameters
   subsection Stokes solver parameters
     set Number of cheap Stokes solver steps = 0
   end
-end
\ No newline at end of file
+end
```

Comment:
:-)

Reply:
sorry, that must have been from `make indent`

Reply:
No, adding the endline is the right thing to do -- your patch did what should have happened when that file was first added :-)